docs/references/sampling_params.md

```diff
@@ -179,25 +184,37 @@ print(response.json())
 The `image_data` can be a file name, a URL, or a base64 encoded string. See also `python/sglang/srt/utils.py:load_image`.
 Streaming is supported in a similar manner as [above](#streaming).
 
-### Structured decoding (JSON, Regex)
-You can specify a JSON schema or a regular expression to constrain the model output. The model output will be guaranteed to follow the given constraints.
+### Structured Outputs (JSON, Regex, EBNF)
+You can specify a JSON schema, Regular Expression or [EBNF](https://en.wikipedia.org/wiki/Extended_Backus%E2%80%93Naur_form) to constrain the model output. The model output will be guaranteed to follow the given constraints. XGrammar uses 
```

Comment:
The tailing "XGrammar uses " is redundant?

Reply:
yes removing it